rules.yaml

```diff
@@ -2,35 +2,41 @@ features:
   - name: api
     scopes:
       - match: module
-        prefix: "modules/{{snake .Module.Name}}_module/{{ snake .Module.Name}}/"
         documents:
+          - source: CMakeLists.txt.tpl
+            target: CMakeLists.txt
+            force: false
           - source: "module/CMakeLists.txt.tpl"
-            target: "CMakeLists.txt"
+            target: "modules/{{snake .Module.Name}}_module/{{snake .Module.Name}}/CMakeLists.txt"
```

Comment:
the path should be moved back to prefix, once the root CMakeLists is on system level